pootle/apps/pootle_app/management/commands/update_tmserver.py

```diff
@@ -200,8 +201,8 @@ def _initialize(self, *args, **options):
         try:
             self.tm_settings = settings.POOTLE_TM_SERVER[options.get('tm')]
         except KeyError:
-            raise CommandError('Specified Translation Memory is not defined '
-                               'in POOTLE_TM_SERVER.')
+            raise CommandError("Translation Memory '%s' is not defined in "
+                               "POOTLE_TM_SERVER setting." % options.get('tm'))
```

Comment:
IMHO it is better to display the user a specific message telling what is wrong than throwing a traceback that means nothing to the user.